packages/deployer/src/server/handlers/client.ts

```diff
@@ -34,3 +35,18 @@ export function handleTriggerClientsRefresh(c: Context) {
   });
   return c.json({ success: true, clients: clients.size });
 }
+
+// Functions to control hot reload during template installation
+export function disableHotReload() {
+  isTemplateInstallationActive = true;
```

Comment:
I think we should make this more generalized. 
```suggestion
  isHotReadloadDisabled = true;
```